dsl/camel-yaml-dsl/camel-yaml-dsl/src/generated/resources/schema/camelYamlDsl.json

```diff
@@ -7267,6 +7267,100 @@
         } ],
         "required" : [ "uri" ]
       },
+      "org.apache.camel.model.TokenizerDefinition" : {
+        "title" : "Specialized tokenizer for AI applications",
+        "description" : "Represents a Camel tokenizer for AI.",
+        "type" : "object",
+        "additionalProperties" : false,
+        "properties" : {
+          "description" : {
+            "type" : "string",
+            "title" : "Description",
+            "description" : "Sets the description of this node"
+          },
+          "disabled" : {
+            "type" : "boolean",
+            "title" : "Disabled",
+            "description" : "Whether to disable this EIP from the route during build time. Once an EIP has been disabled then it cannot be enabled later at runtime."
+          },
+          "id" : {
+            "type" : "string",
+            "title" : "Id",
+            "description" : "Sets the id of this node"
+          },
+          "langChain4jCharacterTokenizer" : { },
```

Comment:
This looks a bit suspesious that these are here and empty

Reply:
Hm ... interesting (maybe that's why we are not seeing them on the dropdown, @lordrip). 

Thanks @davsclaus, let me investigate this.

Reply:
@davsclaus upon closer inspection this seems fine. We have the same thing for the Marshal definition (which was what I based myself for adjusting this): https://github.com/apache/camel/blob/db7a2bb33eb9d2501134ca9b393d7281d7a1c56f/dsl/camel-yaml-dsl/camel-yaml-dsl/src/generated/resources/schema/camelYamlDsl.json#L3251-L3257

Reply:
okay thanks